.github/workflows/scheduled-scan.yaml

```diff
@@ -10,18 +10,43 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      - name: Run Trivy vulnerability scanner
+      - name: Run Trivy filesystem vulnerability scanner
         id: scan
         uses: aquasecurity/trivy-action@0.7.1
         with:
           scan-type: 'fs'
           format: 'sarif'
-          output: 'trivy-results.sarif'
+          output: 'trivy-filesystem-results.sarif'
           exit-code: '1'
           ignore-unfixed: true
           severity: 'CRITICAL,HIGH'
-      - name: Upload Trivy scan results to GitHub Security tab
+      - name: Upload Trivy filesystem scan results to GitHub Security tab
         uses: github/codeql-action/upload-sarif@v2
         if: always()
         with:
-          sarif_file: 'trivy-results.sarif'
+          sarif_file: 'trivy-filesystem-results.sarif'
+
+  scan-image:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up docker
+        uses: docker/setup-buildx-action@v2
```

Comment:
I don't think you need this step anymore. Docker should be available by default on the GitHub actions runner.

Reply:
Good catch. Solved.